logstash-core/spec/logstash/acked_queue_concurrent_stress_spec.rb

```diff
@@ -0,0 +1,296 @@
+# encoding: utf-8
+require "logstash/util/wrapped_acked_queue"
+require "logstash/event"
+require "logstash/instrument/namespaced_metric"
+
+RSpec.configure do |config|
+  config.backtrace_exclusion_patterns << /rakelib\/test/
+  config.backtrace_exclusion_patterns << /vendor\/bundle/
+end
```

Comment:
It will impact tests after the `acked_queue_concurrent_stress_spec.rb` is loaded even if they are not run.
Should this be in the `spec_helper`?